server/tests/api/test_rounds.py

```diff
@@ -83,6 +83,67 @@ def test_rounds_create_one(
     assert sorted(sampled_jurisdictions) == sorted(jurisdiction_ids[:2])
 
 
+def test_multiwinner_rounds_create_one(
```

Comment:
the pattern i've been using for special concerns like this one has been to make a new test file (e.g. `test_multiple_targeted_contests.py`. that way you can group all the tests for that special concern together and override the existing test fixtures (so you don't have to create a bunch of new fixture names). i'm hoping it will keep the tests easier to navigate and understand. what do you think about moving these new tests and fixtures to one file together?

Reply:
Sure, that's reasonable.